drakrun/drakrun/scripts/cfg.template

```diff
@@ -24,4 +29,4 @@ cpuid="host,htt=0"
 vga="stdvga"
 vif = [ 'type=ioemu,model=e1000,bridge=drak{{ VM_ID }}' ]
 disk = [ {{ DISKS }} ]
-processor_trace_buf_kb = 65536
+processor_trace_buf_kb = 8192
```

Comment:
Just curious, what does this value do?

Reply:
This is reserved for future - Intel Processor Trace support. Right now it's a no-op until you have approproate Xen version.

Reply:
Ah, okay thanks, will read more about `Intel Processor Trace support`